### PR TITLE
DOCSP-36193: Update Admin API Trigger endpoints with EventBridge error handler properties

### PR DESCRIPTION
**Pull request by cbullinger**
## Pull Request Info

Jira ticket: https://jira.mongodb.org/browse/DOCSP-36193

- [App Services Admin API](https://preview-mongodbcbullinger.gatsbyjs.io/atlas-app-services/docsp-36194-eventbridge-admin-api/admin/api/v3/#tag/triggers)  (reference page): Add new `error_handler` object to the Database Trigger type for all relevant Trigger endpoints.

### Reminder Checklist

Before merging your PR, make sure to check a few things.

- [ **N/A** ] Did you tag pages appropriately? 
  - genre
  - programming_language
  - meta.keywords
  - meta.description
- [x] Describe your PR's changes in the Release Notes section
- [x] Create a Jira ticket for related docs-realm work, if any

### Release Notes

- **Reference**
  - App Services Admin API: Update page to support AWS EventBridge custom error handling feature by adding the new `error_handler` object property to Database type Trigger payloads in all relevant Trigger endpoints. 

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

**Review by lindseymoore — APPROVED** [review]
Thanks, LGTM!

File: source/openapi-admin-v3.yaml [hunk 1]
```diff
@@ -7109,6 +7109,19 @@ components:
                     can process events in parallel. If `false`, event
                     ordering is enabled and the Trigger executes events
                     serially.
+            error_handler:
+              type: object
+              description: An object that defines custom error handling for an AWS EventBridge trigger. Valid only for `AWS_EVENTBRIDGE` event processors.
```
Comment (tcannon91) [inline source/openapi-admin-v3.yaml:7114]: Is it worth adding a note (or adjusting the current note) that this is only valid for Database triggers as well?

Currently we support AWS EventBridge processors on all of our trigger types (Database, scheduled, auth), but that isn't out long term plan -- we plan to only support eventbridge for Database triggers eventually. So with that future in mind, we only implemented support for Custom Error Handlers with Database triggers.

File: source/openapi-admin-v3.yaml [hunk 2]
```diff
@@ -7109,6 +7109,19 @@ components:
...
+              properties:
+                config:
+                  type: object
+                  properties:
+                    enabled:
+                      type: boolean
+                      description: If 'true', custom error handling is enabled for the AWS EventBridge trigger. 
+                    function_id:
+                      type: string
+                      description: The ID of the custom error handler invoked when the AWS EventBridge trigger fails and cannot be successfully retried.
```
Comment (tcannon91) [inline source/openapi-admin-v3.yaml:7124]: Just a thought -- custom error handlers are just functions at the end of the day. Would it be more clear to reword this to something closer to this:
```suggestion
                      description: The ID of the function invoked when the AWS EventBridge trigger fails and cannot be successfully retried.
```
If it isn't confusing due to the structure of the object or if we are trying to communicate something by using `custom error handler` then I am on board :) My suggestion is to hopefully alleviate any confusion that they need to look for custom error handler in the UI/app config rather than just any function they have created.

File: source/openapi-admin-v3.yaml (hunk 1 above)
Comment (cbullinger) [reply source/openapi-admin-v3.yaml:7114]: This object is only defined in the database type schema, but it wouldn't hurt to reiterate it here, as well. I'll update

Also, I meant to ask: is this programmatically enforced as invalid for the other trigger types?

File: source/openapi-admin-v3.yaml (hunk 2 above)
Comment (cbullinger) [reply source/openapi-admin-v3.yaml:7124]: No, but I was trying to be consistent with the naming. Agree it's potentially confusing here.

File: source/openapi-admin-v3.yaml (hunk 1 above)
Comment (tcannon91) [reply source/openapi-admin-v3.yaml]: Yes, it is enforced by the api
